qgis_hub_plugin/gui/resource_browser.py

```diff
@@ -34,6 +33,21 @@
 )[0]
 
 
+def store_settings(key_value, stored_value):
```

Comment:
Put `store_setting` and `read_setting` to a different file. common/setting.py might be a good candidate.

Try to separate the code so no file or class is too big.